Module 1 homework task.

dmnikitin opened the pull request:
All evaluation criteria points were met: necessary components, interfaces & classes created, project runs on local env with "ng serve" command.

**NB: default 'ng build' command creates index.html file with incorrect href in HTML `<base>` tag, 
so for building the project please instead use "ng build --base-href ./" or "npm run build".

KonanMentor commented:
@dmnikitin please fix linter issues according to configured `tslint.json`: `npm run lint`

dmnikitin commented:
@KonanMentor thanks! I've fixed linter errors& warnings (all of them were located in *.spec files and initial angular project files, which I forgot to check before submitting:(, 
Also I updated "ng lint" command in angular.json - now by default it uses ESLint.
